lib/shopify-cli/tasks/update_dashboard_urls.rb

```diff
@@ -0,0 +1,43 @@
+module ShopifyCli
+  module Tasks
+    class UpdateDashboardURLS < ShopifyCli::Task
+      NGROK_REGEX = /https:\/\/([a-z0-9\-]+\.ngrok\.io)(.*)/
+
+      def call(ctx, url:)
+        @ctx = ctx
+        project = ShopifyCli::Project.current
+        app_type = project.app_type
+        api_key = project.env.api_key
+        result = Helpers::PartnersAPI.query(ctx, 'get_app_urls', apiKey: api_key)
+        callback = app_type.callback_url
+        app = result['data']['app']
+        app_url = check_application_url(app['applicationUrl'], url)
```

Comment:
we should rename this `app_url` as it's now a boolean. something like `consent` or `confirm`